Eos/GammaLaw/EOS.H

```diff
@@ -151,6 +161,15 @@ Y2X(amrex::Real Y[], amrex::Real X[])
     X[n] = Y[n];
 }
 
+AMREX_GPU_HOST_DEVICE
+AMREX_FORCE_INLINE
+void
+X2Y(amrex::Real X[], amrex::Real Y[])
+{
+  for (int n = 0; n < NUM_SPECIES; n++)
+    Y[n] = X[n];
+}
+
```

Comment:
Hum... Excuse me ? What is that ?

Reply:
Gamma law gases don't have molecular weights, so the conversion is nonsensical.  This just ensures that if Sum(X)=1, so does Sum(Y), and preserves all the things that should be preserved.

Reply:
well there still is a law to convert, X/Xk ends up being 1 but it's still supposed to be valid... But I understand, I did not see this was for the GammaLaw so I guess we're good